etcdmain/etcd.go

```diff
@@ -284,12 +284,12 @@ func startEtcdOrProxyV2() {
 	notifySystemd(lg)
 
 	select {
-	case lerr := <-errc:
+	case err := <-errc:
 		// fatal out on listener errors
 		if lg != nil {
 			lg.Fatal("listener failed", zap.Error(err))
```

Comment:
Nit, might be a good idea to keep the name `lerr` as it is and just change this to `lerr` as this is more readable as listener error. Thanks!

Reply:
thanks for your advice. Although there is no unified form, but keep ` lerr ` is better.